src/js/media-container.js

```diff
@@ -599,8 +599,9 @@ class MediaContainer extends globalThis.HTMLElement {
   }
 }
 
-// NOTE: Since MediaContainer is not currently designed to be used directly as
-// a custom element, unlike other components, we do not `define()` it.
+if (!globalThis.customElements.get('media-container')) {
```

Comment:
This can be postponed for a separate PR or omitted. Tl;dr - this allows folks to use `<media-container>` without all of the state mgmt crud from `<media-controller>` (relevant for splitting out the `MediaStore`). It's not a pre-req for this effort though, so we can def back this out if you'd prefer.